Extensions/TweenBehavior/tweenruntimebehavior.js

```diff
@@ -0,0 +1,288 @@
+/**
+ * @memberof gdjs
+ * @class tween
+ * @static
+ * @private
+ */
+
+gdjs.TweenRuntimeBehavior = function (runtimeScene, behaviorData, owner) {
+  gdjs.RuntimeBehavior.call(this, runtimeScene, behaviorData, owner);
+
+  /** @type Object.<string, gdjs.TweenRuntimeBehavior.TweenInstance > */
+  this._tweens = {};
+};
+
+gdjs.TweenRuntimeBehavior.prototype = Object.create(
+  gdjs.RuntimeBehavior.prototype
+);
+
+gdjs.TweenRuntimeBehavior.thisIsARuntimeBehaviorConstructor =
+  "Tween::TweenBehavior";
+
+gdjs.TweenRuntimeBehavior.TweenInstance = function (instance, hasFinished, progress, currentValue) {
+  this.instance = instance;
+  this.hasFinished = hasFinished;
+  this.progress = progress;
+  this.currentValue = currentValue;
+  this.removeObjectFromScene = false;
+};
+
+gdjs.TweenRuntimeBehavior.ObjectVariableTweenInstance = function (instance, hasFinished, progress, variable, currentValue) {
+  gdjs.TweenRuntimeBehavior.TweenInstance.call(this, instance, hasFinished, progress);
+
+  this.variable = variable;
+  this.currentValue = currentValue;
+}
+
+gdjs.TweenRuntimeBehavior.ObjectPositionTweenInstance = function (instance, hasFinished, progress, x, y) {
+  gdjs.TweenRuntimeBehavior.TweenInstance.call(this, instance, hasFinished, progress);
+
+  this.x = x;
+  this.y = y;
+}
+
+gdjs.TweenRuntimeBehavior.prototype._addObjectVariableTween = (identifier, instance, variable, currentValue) => {
```

Comment:
Replace all the arrow functions => in the file by `function` for 1) compatibility with older browser 2) because it won't work as you think here (`this` will refer to the global object, i.e: `window`).
See my other comment